addons/website_event_track_live/__manifest__.py

```diff
@@ -22,6 +22,9 @@
     ],
     'installable': True,
     'assets': {
+        'web.assets_backend': [
+            'website_event_track_live/static/src/js/website_preview.js',
```

Comment:
So this is the "front end in back end" right?

Reply:
Indeed, the issue happen only when the website preview is present so when we have an url with
`cool-url.com/@/...`
And website_preview is indeed considered backend in the assets :smile: